server/src/main/java/org/apache/iotdb/db/utils/TimeValuePairUtils.java

```diff
@@ -107,4 +115,228 @@ public static TimeValuePair getEmptyTimeValuePair(TSDataType dataType) {
         throw new UnsupportedOperationException("Unrecognized datatype: " + dataType);
     }
   }
+
```

Comment:
Is those code duplicated with the `org.apache.iotdb.cluster.utils.PartitionUtils.Intervals` ? if yes, why repeat coding in two places？
Don't repeat yourself.